### DISCUSSION TITLE
How to add translation to strapi

### DISCUSSION THREAD

**Discussion by filipesmedeiros** [strapi/strapi#6103] [Legacy Topics]
Hey! I want to add Portuguese (pt_pt) to strapi's admin dashboard for myself, so might as well share it with everyone. Where can I translate the dashboard and add the files, and then how can I keep updated with new stuff that needs to be translated along the way?

**Comment by robertpiosik** [comment]
Isn't Portuguese already [here](https://github.com/strapi/strapi/blob/master/packages/strapi-admin/admin/src/translations/pt.json)?

**Reply by filipesmedeiros** [reply]
Ok I'm dumb enough to not scroll on the languages dropdown. Can we have a scroll indicator? Ahah

Also, it doesn't seem complete, can I just edit and PR that file?

**Reply by lauriejim** [reply]
Yes, please provide a PR to add / updated missing / incorrect translation for pt_pt.

**Reply by filipesmedeiros** [reply]
It's here! #6022
